src/scripts/index.js

```diff
@@ -84,11 +93,30 @@ requestInfoBody.appendChild(requestInfoBodyForm);
 requestInfoBodyForm.appendChild(textareaLabel);
 requestInfoBodyForm.appendChild(requestInfoBodyTextBox);
 
-CodeMirror.fromTextArea(requestInfoBodyTextBox, {
+const bodyTextArea = CodeMirror.fromTextArea(requestInfoBodyTextBox, {
   lineNumbers: true,
   theme: "yonce",
   mode: 'application/json',
   lint: true,
 });
 
 document.body.appendChild(requestInfo);
+
+// Accepting request with the Enter key
+function handleEnterInInput(e) {
+  if (e.key === "Enter") {
+    handleSendAndDisplayRequest();
+  }
+}
+
+function handleSendAndDisplayRequest() {
+  makeRequestAsync(
+    inputSend.value,
+    selectSend.value,
+    JSON.parse(bodyTextArea.getValue())
+  )
+    .then((response) => {
+      //TODO display `response` object in MIL-5_Response component
```

Comment:
I think the easiest option will be to just add a line of code to display the result of the request when we will finish the Response component, but I am not sure if this is a proper assumption :)